src/scene/export.ts

```diff
@@ -94,11 +94,13 @@ export function exportToSvg(
     <style>
       @font-face {
         font-family: "Virgil";
-        src: url("https://excalidraw.com/FG_Virgil.ttf");
+        src: url("https://excalidraw.com/FG_Virgil.woff2") format("woff2"),
+             url("https://excalidraw.com/FG_Virgil.ttf") format("ttf");
       }
       @font-face {
         font-family: "Cascadia";
-        src: url("https://excalidraw.com/Cascadia.ttf");
+        src: url("https://excalidraw.com/Cascadia.woff2") format("woff2"),
+             url("https://excalidraw.com/Cascadia.ttf") format("ttf");
```

Comment:
Why do you keep both? Is there a reason why a fallback is useful?

Reply:
Yes, was thinking about fallback. But it didn't work for svg file 😔 reverted this change